packages/components/expandablePanel/src/ExpandablePanel.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  ExpandablePanelProps,
+  ExpandablePanelStyledProps,
+} from './expandablePanel.types';
+import { ExpandablePanelStyled } from './expandablePanel.styles';
+import { deepMerge, useTheme } from '@douro-ui/react';
+import { ReactNode, useEffect, useState } from 'react';
+import {
+  ExpandablePanelBody,
+  ExpandablePanelHeader,
+} from './expandablePanelTypes';
+
+const ExpandablePanel = ({
+  styled,
+  disabled,
+  header,
+  children,
+  expanded,
+  startExpanded,
+  ...props
+}: ExpandablePanelProps): ReactNode => {
+  const theme = useTheme();
+  const [isExpanded, setIsExpanded] = useState(startExpanded || false);
```

Comment:
Why not set the default value of `startExpanded` to false?

Reply:
This is no longer part of the component~